lib/src/components/NxForm/NxForm.tsx

```diff
@@ -102,7 +103,8 @@ const _NxForm = forwardRef<HTMLFormElement, Props>(
             { submitMaskState != null &&
               <NxSubmitMask success={submitMaskState}
                             message={submitMaskMessage}
-                            successMessage={submitMaskSuccessMessage} />
+                            successMessage={submitMaskSuccessMessage}
+                            aria-label="form submit message" />
```

Comment:
Would "form submit status" be better?

Reply:
> Would "form submit status" be better?

Yeah that sounds a lot better: e979d54864786a14c94561dfc795826979e8f4d3